packages/extension/src/background/browser-action.js

```diff
@@ -0,0 +1,6 @@
+import { Extension } from '@mujo/utils'
+
+const { tabs } = Extension
+
+export const onBrowserAction = () =>
+  tabs.create({ url: 'chrome://newtab' })
```

Comment:
Can you see if you can open up the path to the index file so we do not need to up date this when we make new tab optional?

https://developer.chrome.com/extensions/runtime#method-getURL

Reply:
Yes, it does open up a new tab.

Reply:
Can you get this to open up a direct reference to the index.html file?